models/arbswap/arbswap_trades.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(
+        alias ='trades',
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "arbswap",
+                                \'["chrispearcx"]\') }}'
+        )
+}}
+
+{% set arbswap_models = [
+'arbswap_arbitrum_trades'
```

Comment:
In order for arbswap.trades to work, we need to implement `arbswap_arbitrum_trades`.